CHANGELOG.md

```diff
@@ -52,6 +52,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2611](https://github.com/Pycord-Development/pycord/pull/2611))
 - Fixed `TypeError` when passing `skus` parameter in `Client.entitlements()`.
   ([#2627](https://github.com/Pycord-Development/pycord/issues/2627))
+- Fixed `Webhook._WebhookState` missing `store_poll` method.
```

Comment:
saying something like fixed attribute error when sending polls with a webhook or similar should work. Changelogs should be descriptive of what is changed at a high level instead of the details.